rerun

```diff
@@ -482,21 +482,53 @@ rerun_options_parse() {
 # 
 rerun_modules() {
     (( ! $# == 1 )) && { 
-	    rerun_die 'wrong # args: should be: rerun_modules directory'
+	    rerun_die 'wrong # args: should be: rerun_modules path'
     }
-    [[ ! -d $1 ]] && rerun_die "directory not found: $1"
     local -a modules
-    for f in $1/*/metadata
+    for dir in $(rerun_module_path_elements $1)
     do
-       if [[ -f $f ]]
-        then 
-            local module="$(basename $(dirname $f))" 
-            [[ -z "${modules:-}" ]] && modules=( $module ) || modules=( ${modules[*]} $module )
-        fi 
+        [[ ! -d $dir ]] && rerun_die "directory not found: $dir"
+        for f in $dir/*/metadata
+        do
+            if [[ -f $f ]]
+            then 
+                local module="$(basename $(dirname $f))" 
+                [[ -z "${modules:-}" ]] && modules=( $module ) || modules=( ${modules[*]} $module )
+            fi 
+        done
+    done
+    echo ${modules[*]} | sort | uniq
```

Comment:
I think the `| sort | uniq` may not do anything. The input is an array, not multi-line.And anyway there is no problem if